tests/neo.UnitTests/Network/P2P/Payloads/UT_Transaction.cs

```diff
@@ -1254,77 +1254,7 @@ public void Test_VerifyStateDependent()
                 Assert.IsTrue(data.Completed);
 
                 tx.Witnesses = data.GetWitnesses();
-                tx.VerifyStateDependent(snapshot, new TransactionVerificationContext()).Should().Be(VerifyResult.Succeed);
-            }
-        }
-
-        [TestMethod]
-        public void Test_Verify()
-        {
-            var snapshot = TestBlockchain.GetTestSnapshot();
-            var tx = new Transaction()
-            {
-                Attributes = Array.Empty<TransactionAttribute>(),
-                NetworkFee = 0,
-                Nonce = (uint)Environment.TickCount,
-                Script = new byte[Transaction.MaxTransactionSize],
-                Signers = new Signer[] { new Signer() { Account = UInt160.Zero } },
-                SystemFee = 0,
-                ValidUntilBlock = 0,
-                Version = 0,
-                Witnesses = new Witness[0],
-            };
-            tx.Verify(snapshot, new TransactionVerificationContext()).Should().Be(VerifyResult.Invalid);
-
-            var walletA = TestUtils.GenerateTestWallet();
-            var walletB = TestUtils.GenerateTestWallet();
-
-            using (var unlockA = walletA.Unlock("123"))
-            using (var unlockB = walletB.Unlock("123"))
-            {
-                var a = walletA.CreateAccount();
-                var b = walletB.CreateAccount();
-
-                var multiSignContract = Contract.CreateMultiSigContract(2,
```

Comment:
Why this test was removed?

Reply:
Because `Transaction.Verify` method is removed.

Reply:
But we should call the other two methods instead of remove the multisign verification

Reply:
We have UT for other two methods already.